docker-for-mac/release-notes.md

```diff
@@ -13,7 +13,54 @@ This page contains information about the new features, improvements, known issue
 For information about Edge releases, see the [Edge release notes](edge-release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
-## Stable Releases of 2019
+## Docker Desktop Community 2.2.0.0
+
+> [Download](https://hub.docker.com/?overlay=onboarding)
+>
+> You must sign in to Docker Hub to download Docker Desktop.
+
```

Comment:
Shouldn’t we have a 
```
Docker Desktop 2.2.0.0 contains a Kubernetes upgrade. Note that your local Kubernetes cluster will be reset after installing this version.
```
?

Reply:
Added in https://github.com/docker/docker.github.io/pull/10171